x-pack/legacy/plugins/security/public/views/management/role_mappings/model/except_any_rule.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { RuleGroup } from './rule_group';
+import { Rule } from './rule';
+import { ExceptAllRule } from './except_all_rule';
+import { ExceptFieldRule } from './except_field_rule';
+
+/**
+ * Represents a group of rules in which none can evaluate to true (all must evaluate to false).
+ */
+export class ExceptAnyRule extends RuleGroup {
+  constructor(private rules: Rule[] = []) {
+    super();
+  }
+
+  /** {@see RuleGroup.getRules} */
+  public getRules() {
+    return [...this.rules];
+  }
+
+  /** {@see RuleGroup.getDisplayTitle} */
+  public getDisplayTitle() {
+    return i18n.translate('xpack.security.management.editRoleMapping.exceptAnyRule.displayTitle', {
+      defaultMessage: 'None are true',
```

Comment:
Nit: the other messages are "All are true", "Any are true", and "Any are false". I think "All are false" would be more consistent here.

Reply:
Good suggestion, changed!